Projects/Blinky/Blinky.c

```diff
@@ -28,6 +29,8 @@
 static osThreadId_t tid_thrLED;         // Thread id of thread: LED
 static osThreadId_t tid_thrButton;      // Thread id of thread: Button
 
+static __NO_RETURN void app_main (void *argument);
```

Comment:
Small inconsistency regarding order "static" and "__NO_RETURN" in app_main vs. thrLED and thrButton

Reply:
I suggest specifying __NO_RETURN first.

Reply:
Actually checking with MISRA it seems that static should be first.